assets/css/styles.css

```diff
@@ -0,0 +1,283 @@
+/* =============================================
+ *
+ *   FIXED RESPONSIVE NAV
+ *
+ *   (c) 2014 @adtileHQ
+ *   http://www.adtile.me
+ *   http://twitter.com/adtilehq
+ *
+ *   Free to use under the MIT License.
+ *
+ * ============================================= */
+ ---
+ ---
+ @import "jekyll-theme-midnight";
```

Comment:
Does this belong here? Did the library include this? This looks like an scss-statefemt (the one we have in `style.scss`).

Reply:
I think this file is also copied. I need to ask @benszau he added the style depending files for the bar...